scripts/styles.css

```diff
@@ -1,9 +1,82 @@
+body {
+  max-width: 800px;
+  margin: auto;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 18px;
+}
+
+ul {
+  padding: 0;
+}
+
+h1 {
+  text-align: center;
+}
+
+.books-list {
+  border: 2px solid black;
+}
+
+.books-list li:nth-child(odd) {
+  background: #ddd;
+}
+
+button {
+  border: 1px solid black;
+  font-size: 15px;
+}
+
+button:enabled {
+  border: 1px solid black;
+  margin: 5px;
+  box-shadow: 3px 3px 0 1px black;
+  font-size: 15px;
+}
+
 .bookEntry {
-  border-bottom: 1px solid grey;
   padding: 5px;
   list-style: none;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
 }
 
 .addForm {
-  margin-top: 30px;
+  margin: auto auto;
+  width: 250px;
+  display: flex;
```

Comment:
- [ ] [OPTIONAL] Let's strive to keep the code DRY. This property:
```CSS
display: flex
```
has been repeated in the CSS. Please refactor it to a class and apply the class in the HTML file to elements that share the same display property.

### Example code
```CSS
.d-flex {
  display: flex;
  align-items: center;
}
```

Then apply this class to HTML element that needs this behavior:

```HTML
  <section class="d-flex">
```